OWNERS_ALIASES

```diff
@@ -1,7 +1,7 @@
 # Sort the member alphabetically.
 aliases:
   sig-critical-approvers-cdc:
-    - kevin-xianliu
+    - yudongusa
```

Comment:
@yudongusa

PTAL, is we should add other members, please review with suggestion code.